src/Components/Settings/AdminComponent.jsx

```diff
@@ -77,8 +79,20 @@ const AdminComponent = () => {
                     Toast.success({
                         text: "Admin Removed Successfully",
                     });
+
+                    let removedUser = adminData?.filter((user) => user.userId == userId);
+                    console.log(removedUser)
                     const data = adminData?.filter((user) => user.userId != userId);
                     setAdminData(data);
+
+                    //////////////////////////
```

Comment:
Role deleted

Reply:
We are going to deactivate the user when the "owner" deletes the user by calling this function from `generalFunction`

```
    // Function to deactivate user
    deactivateUser: async (UserID) => {
        const { data, error } = await supabase
            .from('users')
            .update({is_active: false})
            .eq('id', UserID);
        if (error) {
            throw error;
        }
        return data;
    },
    
    ```